### ISSUE TITLE
Ishikawa diagram

### ISSUE THREAD

**Issue by DanielLipowicz** [mermaid-js/mermaid#4784]
### Proposal

Mermaid.js should add Ishikawa diagram because it's seems to be low hanging fruit. Implantation of this diagram could help to analyze and solve problems which teams are struggling with.  Construction of this diagram is very similar to mind map but with different layout.
https://en.wikipedia.org/wiki/Ishikawa_diagram

### Use Cases

When team are struggling with any complex problem with different possible source. They are able to divide it to various smaller categories and analyze root case of issue. 

Following Wikipedia:
The 8 Ps (used in product marketing)
This common model for identifying crucial attributes for planning in product marketing is often also used in root-cause analysis as categories for the Ishikawa diagram:

Product (or service)
Price
Place
Promotion
People (personnel)
Process
Physical evidence (proof)
Performance

### Screenshots

![image](https://github.com/mermaid-js/mermaid/assets/16608863/3006aa73-8c38-401e-87ea-9d0b96450aa4)

### Syntax

As same as on mind map. There is nothing to change there.

### Implementation

This is a proposal which I'd love to see built into mermaid by the wonderful community.

**Comment by adriens** [comment]
Thatcher would be awesome

**Comment by Francis-Beaulieu** [comment]
This is a great idea!

**Comment by adriens** [comment]
Pleine make it happen

**Comment by codering** [comment]
I like it.

**Comment by estevan83** [comment]
I like it.

**Comment by micrax** [comment]
Yes, it's a very good idea to complete Mermaid. More people will use Mermaid with this feature.

**Comment by suthep-cto** [comment]
wait for it!

**Comment by adriens** [comment]
> wait for it!

We do :star_struck:

**Comment by Roba1993** [comment]
We directly would use Mermaid when this feature is available

**Comment by T9xz** [comment]
very underrated chart

**Comment by matthewww** [comment]
This would be great. Plus, mermaids have fishbones (right?).

**Comment by samim23** [comment]
want this badly

**Comment by JmiXIII** [comment]
Would awesome !

**Comment by emanuelegissi** [comment]
Would be a great addition!

**Comment by mtbirdsall-alianza** [comment]
Is this getting any traction? We're trying to decide on internal diagramming tools and mermaid is in the lead minus this feature.

**Comment by ChristofferCOASD** [comment]
+1

**Comment by ziidonato** [comment]
+1

**Comment by joaquin1115** [comment]
+1

**Comment by Swiddis** [comment]
I wonder if this could be implemented relatively lightweight by adding a layout to the existing [mindmap system](https://mermaid.js.org/syntax/mindmap.html). Just something that forces it to expand out to the left and follow the correct angles. The indentation-based syntax is pretty much exactly what we need for actually writing the content.

(Relatedly, as a workaround I'm using Mindmaps since my use case doesn't strictly require the Ishikawa layout)

**Comment by danieldirks1** [comment]
+1

**Comment by danieldirks1** [comment]
+1

**Comment by estevan83** [comment]
+1

**Comment by SuperOuxx** [comment]
Mermaid is an AI-friendly tool. If mermaid implements the Ishikawa diagram, it can perform more complex analysis through AI, such as software requirements analysis.

**Comment by rrayhka** [comment]
+1

**Comment by ccivit** [comment]
I would love to see this feature too

**Comment by dryor** [comment]
+1

**Comment by IuryChagas25** [comment]
+1

**Comment by jzhouw** [comment]
+1

**Comment by SugarRayLua** [comment]
+1 :-)

**Comment by gautierchomel** [comment]
+1

**Comment by juliusHin** [comment]
+1

**Comment by saparhadi** [comment]
+1

**Comment by ChristofferCOASD** [comment]
For anyone reaching this issue again, @exoego made an implementation in #7387 that you can use in the `develop` https://develop.git.mermaid.live/ editor:


<img width="1108" height="934" alt="Image" src="https://github.com/user-attachments/assets/4180223d-5064-4937-98ec-f221722e06ae" />
